src/FLEx-ChorusPluginTests/FLEx-ChorusPluginTests.csproj

```diff
@@ -1,263 +1,275 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't think this is true. For it to work from Visual Studio the package.config file needs to be in the solution somewhere.